### PR TITLE
Hg 846 - moving to shadow DOM for ember compatibility 

### PR DESCRIPTION
**Pull request by lizlux**
We were having some issues with polymer components when transitioning between views that had the component and views that didn't. Per https://github.com/emberjs/ember.js/issues/11361, we've switched to using shadow DOM and this seems to have fixed the issue.

### CODE REVIEW

File: src/components/top-bar/top-bar.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
 				align-items: center;
```
Comment (lizlux) [inline src/components/top-bar/top-bar.html:28]: Here's a page that was actually edited.

File: src/components/user-status/user-status.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
```
Comment (lizlux) [inline src/components/user-status/user-status.html:8]: Here's another page

File: gh-pages/_includes/head.html [hunk 3]
```diff
@@ -12,6 +12,10 @@
 	<link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
 
 	<script src="{{ '/vendor/wikia-style-guide/dist/vendor/webcomponentsjs/webcomponents-lite.js' | prepend: site.baseurl }}"></script>
```
Comment (lizlux) [inline gh-pages/_includes/head.html:14]: here's another place that was updated.

File: gh-pages/_includes/global-nav.html [hunk 4]
```diff
@@ -5,7 +5,7 @@
 	<a class="menu-icon dropdown-icon left default-content">
 		<img src="{{ site.assets_path | prepend: site.baseurl }}/images/icons/icon_nav.svg">
```
Comment (lizlux) [inline gh-pages/_includes/global-nav.html:6]: This file was updated

**Comment by kenkouot** [comment]
LGTM

**Pull request merged**